tests_integration/runPrettier.js

```diff
@@ -44,9 +44,11 @@ function runPrettier(dir, args, options) {
   const originalArgv = process.argv;
   const originalExitCode = process.exitCode;
   const originalStdinIsTTY = process.stdin.isTTY;
+  const originalStdoutIsTTY = process.stdout.isTTY;
 
   process.chdir(normalizeDir(dir));
   process.stdin.isTTY = !!options.isTTY;
+  process.stdout.isTTY = !!options.stdoutIsTTY;
```

Comment:
I think we can just use the existing `isTTY` here; I don't know of any real-world situations where stdin is a tty and stdout isn't (or vice-versa)

Reply:
https://nodejs.org/api/process.html

> ```
> $ node -p "Boolean(process.stdin.isTTY)"
> true
> $ echo "foo" | node -p "Boolean(process.stdin.isTTY)"
> false
> $ node -p "Boolean(process.stdout.isTTY)"
> true
> $ node -p "Boolean(process.stdout.isTTY)" | cat
> false
> ```

I think it is correct to use `process.stdout.isTTY` here.